tc/c2/tc_op.h

```diff
@@ -102,12 +101,8 @@ class TcOp : public Operator<Context> {
         inputVoidPtrs_.push_back(inputDLTensors_.back()->data);
       }
 
-      auto parsedTcs = tc::detail::parse(tc_);
-      CHECK_EQ(parsedTcs.count(tcName_), 1u)
-          << "attempting to access undefined function " << tcName_;
-
-      auto outTensorInfo = tc::detail::inferOutputTensorInfo(
-          parsedTcs.at(tcName_), rawInputDLTensors_);
+      auto outTensorInfo =
+        tc::inferOutputTensorInfo(tc_, tcName_, rawInputDLTensors_);
```

Comment:
independent change?

Reply:
spurious change actually, thanks!

Reply:
my bad, I misunderstood until I saw it in github, please ignore ..